.werft/jobs/build/payment/stripe-configmap.yaml

```diff
@@ -7,7 +7,7 @@ data:
   config: |
     {
       "usageProductPriceIds": {
-        "EUR": "price_1LFZhJGadRXm50o3v27S1DB1",
-        "USD": "price_1LFZhJGadRXm50o3YEAaspXu"
+        "EUR": "price_1LiId7GadRXm50o3OayAS2y4",
+        "USD": "price_1LiIdbGadRXm50o3ylg5S44r"
```

Comment:
What do these mean? 🙃

Reply:
I've created new Prices in our "test" Stripe account, with tax enabled, so that we can test tax things in preview environments. (Our previous Prices were not tax compatible, because we hadn't enabled Stripe Tax in the "test" account yet.)

Merging this update should unblock work on https://github.com/gitpod-io/gitpod/pull/13002 (which currently fails because it's impossible to subscribe with automatic tax enabled to a price that is not tax compatible) 😊

Reply:
Thanks for the explanation!